plugins/generator-1.19.2/forge-1.19.2/utils/mcitems.ftl

```diff
@@ -109,6 +109,37 @@
     </#if>
 </#function>
 
+<#macro containsAnyOfBlocks elements blockToCheck>

```

Comment:
Now that this file contains a `<#macro>`, it should be named `mcitems.java.ftl` starting 1.20.1 (sounds obvious, but just to highlight it :man_shrugging:).

Reply:
Fair point. However, this will make our lives much harder by having to change this to all files that could be just copied otherwise.

In this case, I think it is better to change this from macro to function the way I did with ingredients @Goldorion and keep mcitems.ftl macro-free

Reply:
But what is the difference a macro and a function, except how we call them?

Reply:
I don't know how you are able to quickly understand this, but for me, the macro was so much easier and faster to understand, but I'm not the boss, so I won't argue 🤷

![image](https://github.com/MCreator/MCreator/assets/38427877/9f062364-f368-4af6-a3fe-472434287a9c)

Reply:
The issue is with mentioned rename of the file which would affect most of the files in generators, so this is lesser of evil